### PR TITLE
MPP-3064 - Email tracker link 

### PR DESCRIPTION
**Pull request by lloan**
This PR fixes MPP-3064

Clicking tracker reports in forwarded emails directs to a white screen noting a console error. 
 

# New feature description

Suggested fix: @Vinnl the email should not include a link to the tracker report if no trackers were detected. And now that we've already sent these emails to folks, we probably also want to update the tracker report page to deal with 0 detected trackers.

This is a two part fix. Update the email template (wrapper) and update the tracker report page. 

We updated the email template to only render an anchor tag if trackers were detected, additional check added, (link and number of). We also added a check in the tracker report page, to make sure we render the appropriate error message if trackers are not found in the `reportData` object as assumed previously. 

# Screenshot (if applicable)

Not applicable.

# How to test

Use this URL: https://deploy-preview-3410--fx-relay-demo.netlify.app/tracker-report/#%7B%22sender%22:%20%22Microsoft@engage.microsoft.co[%E2%80%A6]t%22:%201683402542065,%20%22trackers%22:%20%7B%7D%7D 

On PRD, this URL would give you an application error page / white screen. Here, it should give you an error message, same as if we were missing `reportData`. 

# Checklist (Definition of Done)
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] I've added or updated relevant docs in the docs/ directory
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] l10n changes have been submitted to the l10n repository, if any.

### CODE REVIEW

**Review by Vinnl — APPROVED** [review]
This change works and the code looks good. It does still result in a full-page error message, so I added some suggestions for more graceful degradation.

Also, there's a `[..]` in the link in the description that breaks it, I think a working one should be https://deploy-preview-3410--fx-relay-demo.netlify.app/tracker-report/#{%22sender%22:%22email@example.com%22,%22received_at%22:1683890758220} (although they have the same end result).

File: frontend/src/pages/tracker-report.page.tsx [hunk 1]
```diff
@@ -48,7 +48,13 @@ const TrackerReport: NextPage = () => {
       </div>
     );
   }
-  if (reportData === null) {
+
+  // check if reportData is null and check if trackers are set.
+  if (
+    reportData === null ||
+    !reportData.trackers ||
+    Object.keys(reportData.trackers).length === 0
```
[the remark this answers is not in the document]
Comment (lloan) [reply frontend/src/pages/tracker-report.page.tsx]: changed

**Review by Vinnl — APPROVED** [review]
Awesome, here's another approval!

**Comment by groovecoder** [comment]
Any more reasons to wait on this? Or can we update the branch and merge it?

**Pull request merged**